routes/items.js

```diff
@@ -4,6 +4,7 @@ const router = express.Router()
 const { allItemsQuery, filteredItemsQuery, respondWithItems } = require( './items/item_response' )
 const { buildTree } = require( './items/tree_creation' )
 const findAllItems = require('./items/find_all_items')
+const grabUserStats = require('../src/weekly_report')
```

Comment:
Please use a more meaningful name - it feels odd to me that a module that is called `weekly_report` exports a method called `grabUserStats`.  How about `weeklyUserReport` or some such?